addons/html_editor/static/src/editor/core/delete_plugin.js

```diff
@@ -38,8 +40,6 @@ import {
     rightPos,
 } from "../utils/position";
 import { getDeepRange, preserveCursor, setSelection } from "../utils/selection";
```

Comment:
you keep the import?

Reply:
I need it for an utils in the file. We should maybe move the utils in the plugin